frontend/plugins/frontline_ui/src/widgets/automations/modules/facebook/components/bots/components/automationBotsRecordTable.tsx

```
@@ -0,0 +1,27 @@
+import { RecordTable } from 'erxes-ui';
+import { AutomationBotSheetForm } from '~/widgets/automations/modules/facebook/components/bots/components/AutomationBotSheetForm';
+import { automationFacebookBotsColumns } from '~/widgets/automations/modules/facebook/components/bots/components/automationFacebookBotsColumns';
+import { useAutomationBotsRecordTable } from '~/widgets/automations/modules/facebook/components/bots/hooks/useAutomationBotsRecordTable';
+
+export const AutomationBotsRecordTable = ({}: any) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)